src/shared/components/hero/Hero.md

```diff
@@ -42,3 +42,31 @@ Changed from 20vh to 35vh in version 6.1.4
     <div class="hw-hero__image"></div>
   </div>
 ```
+
+### Hero with content
+Content can be placed on top of the hero image and positioned right, left or center, and top, bottom or middle
+
+```code
+[Top]           .hw-hero__content-top
```

Comment:
Should say .hw-hero__content--top, with two dashes to denote a modifier.
Goes for all of these